backend/tests/api/v1/endpoints/test_chatbot.py

```
@@ -32,37 +25,27 @@ async def test_chatbot_health_unauthenticated(client):
-    app.dependency_overrides[get_current_user] = override_get_current_user
-    try:
-        with patch(
-            "util.session_id.session_id_management", return_value="test_session_id"
-        ), patch("services.chat_service.get_answer") as mock_get_answer, patch(
-            "util.chatbot.save_chat_log"
-        ) as mock_save_log:
+    with patch(
+        "util.session_id.session_id_management", return_value="test_session_id"
+    ), patch("services.chat_service.get_answer") as mock_get_answer, patch(
+        "util.chatbot.save_chat_log"
+    ) as mock_save_log:
```

Ruff (0.14.5)

Local variable `mock_save_log` is assigned to but never used

Remove assignment to unused variable `mock_save_log`

(F841)